src/structures/Interaction.js

```diff
@@ -67,6 +68,12 @@ class Interaction extends Base {
      * @type {number}
      */
     this.version = data.version;
+
+    /**
+     * This member permissions, if exists, in the channel this interaction was sent in
+     * @type {?Permissions}
+     */
+     this.memberPermissions = data.member?.permissions ? new Permissions( data.member.permissions ) : null;
```

Comment:
Prettier please, also update typings

Reply:
Actually, I believe data.member.permissions can be 0, so this should be a typeof or an in check instead

Reply:
Good catch. The possible 0 permission was also accounted for. 👍